2000-10-20  Jeffrey Oldham  <oldham@codesourcery.com>

	* decl.c (revert_static_member_fn): Fixed typo.

--- gcc/cp/decl.c	2000/10/18 18:59:03	1.699
+++ gcc/cp/decl.c	2000/10/20 16:06:05	1.700
@@ -14603,7 +14612,7 @@ revert_static_member_fn (decl)
   if (CP_TYPE_QUALS (TREE_TYPE (TREE_VALUE (args)))
       != TYPE_UNQUALIFIED)
     cp_error ("static member function `%#D' declared with type qualifiers",
-	      *decl);
+	      decl);
 
   args = TREE_CHAIN (args);
   tmp = build_function_type (TREE_TYPE (function), args);
